mpitest/test_mpi_opt.py

```diff
@@ -155,6 +156,46 @@ def test_parab_subbed_Pcomps(self):
         if not MPI or self.comm.rank == 1:
             assert_rel_error(self, model['par.s2.p.x'], 3.0, 1.e-6)
 
+
```

Comment:
that's the test I was working on that you said wasn't a priority, so I just commented it out so I could go back to it later